plugins/qlcplus_4.12.1/export.js

```diff
@@ -87,12 +87,25 @@ module.exports.export = async function exportQlcPlus(fixtures, options) {
 
   // add gobo images not included in QLC+ to exported files
   Object.entries(customGobos).forEach(([qlcplusResName, oflResource]) => {
-    // oflResource.image is in the form "data:<mimeType>;charset=utf-8;base64,<base64Data>"
-    const [, mimeType,,, base64Data] = oflResource.image.split(/[:;,]/);
+    // oflResource.image is in one of the following forms:
+    // "data:<mimeType>;base64,<base64Data>" or
+    // "data:<mimeType>;charset=utf-8,<uriEncodedData>"
+    const [mediaType, imageData] = oflResource.image.replace(`data:`, ``).split(`,`);
```

Comment:
Please extract this to the Resource class, which should store mimeType and base64Data and has a convenience getter to create the dataUrl